documentation/site/content/samples/azure-kubernetes-service/model-in-image.md

```diff
@@ -79,7 +105,7 @@ helm repo add weblogic-operator https://oracle.github.io/weblogic-kubernetes-ope
 ```
 $ helm install weblogic-operator weblogic-operator/weblogic-operator \
   --namespace sample-weblogic-operator-ns \
-  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:4.0.6 \
+  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:4.1.7 \
```

Comment:
If the set image parameter is omitted, then you will get the latest.  

helm search repo weblogic-operator --versions returns all the available versions (be sure update repo first)

helm install ... --version <version> will install the particular version from the repo.

Reply:
Microsoft certifies a specific set of well tested versions for this guidance, see https://github.com/oracle/weblogic-azure/blob/main/weblogic-azure-aks/src/main/resources/weblogic_tooling_family.json . We will update this json file to the appropriate version when the PR is merged.

So, we specifically do not want "latest", at least for this, azure specific,  guidance.

Reply:
No, what I meant is you do not need to specify the entire `--set image ....`  ,  just skip it and you will get the latest and use `--version`  when installing will pick the right helm chart version and the right image version automatically.

Reply:
Fixed in 7ca8da53c70.